department-of-veterans-affairs/vets-website
# Update EC and CC details pages to work after refresh

## Description

This PR
- Updates EC details page to follow the same pattern as the request details page and fetch a single request when loaded directly
- Updates the CC details page to fetch two years worth of appointments and filter down to the matching id.
- Add some focus and title handling (there's another ticket for this, so it's not complete).

How to test:
- Open http://localhost:3001/health-care/schedule-view-va-appointments/appointments/
- Click on an EC request or CC appointment
- Refresh page
- Verify that page loads with correct appointment info

## Testing done
Local and unit testing

## Acceptance criteria
- [ ] Page refresh works

## Definition of done
